.werft/build.yaml

```diff
@@ -241,6 +241,10 @@ pod:
           echo "done" \
         ) | werft log slice "chowning /workspace and $GOCACHE"
 
+        {{ if .Annotations.leewayfromgit }}
+        scripts/install-leeway-from-git.sh "{{ .Annotations.leewayfromgit }}" | werft log slice "Building fresh Leeway binary from source"
```

Comment:
```suggestion
        GOBIN=$(dirname $(which leeway)) go install github.com/gitpod-io/leeway@latest | werft log slice "Building fresh Leeway binary from source"
```

Reply:
Or pass whatever tag to `@latest`

Reply:
Hm you might need go in sudo path, if leeway is in a restricted path. So this should work:

```bash
GOBIN=$(pwd) go install github.com/gitpod-io/leeway@latest && sudo mv leeway $(dirname $(which leeway))
```